fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -2262,7 +2262,7 @@ explainDesc
     ;
 
 optimizerTrace
-    : TRACE OPTIMIZER
+    : TRACE (OPTIMIZER | REWRITE)
```

Comment:
how about `trace log`?

Reply:
For now, i only add mv rewrite log into output. I doubt trace log is a bit confused?

Reply:
Only MV is fine, we can extend this command later, I have a idea to resolve the question, and I can support it if I'm free

Reply:
Good~